frontend/src/components/session/session_form.jsx

```diff
@@ -5,13 +5,17 @@ class SessionForm extends React.Component {
   constructor(props) {
     super(props);
 
+    const search = props.location.search; // could be '?foo=bar'
+    const params = new URLSearchParams(search);
+    const foo = params.get('email'); // bar
```

Comment:
can we use a more accurate naming convention here? foo and bar don't really signify anything to someone reading the code